std_msgs/Bool create_new
---
geometry_msgs/Point32[] creeper_locations
std_msgs/Bool reached_end
